lib/scholar/impute/simple_imputer.ex

```diff
@@ -0,0 +1,215 @@
+defmodule Scholar.Impute.SimpleImputer do
+  @moduledoc """
+  Univariate imputer for completing missing values with simple strategies.
```

Comment:
Could we add some examples of how to use it?

Reply:
Sure